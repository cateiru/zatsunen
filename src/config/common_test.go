package config_test

import (
	"testing"

	"github.com/cateiru/zatsunen/src/config"
	"github.com/gkampitakis/go-snaps/snaps"
)

func TestGetCommonConfig(t *testing.T) {
	commonConfig := config.GetCommonConfig()

	snaps.MatchSnapshot(t, commonConfig)
}
