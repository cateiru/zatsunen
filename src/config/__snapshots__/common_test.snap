
[TestGetCommonConfig - 1]
config.CommonConfig{}
---
